### PR TITLE
Fix/755 fixed image missing alt

### PR DESCRIPTION
**Pull request by raiseandfall**

### CODE REVIEW

File: src/components/organisms/CardCertificate/CardCertificate.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    const titleClass = [
+      'buv-c-card__title',
+      hideRecordLink ? 'buv-c-card__title--no-padding' : ''
+    ].join(' ');
+
+    return html`
+      ${CSS}
+      <section class='buv-c-card'>
+        <div class='buv-c-card__img-wrapper'>
+          ${!this.logoIsErroring ? this.organizationLogo(props) : this.organizationName(props)}
```
Comment (lemoustachiste) [inline src/components/organisms/CardCertificate/CardCertificate.js]: wait what's the difference between this and the `alt` text?

Comment (lemoustachiste) [reply src/components/organisms/CardCertificate/CardCertificate.js]: could we handle that from a CSS only perspective? I am a bit reluctant to have this kind of code.

Comment (raiseandfall) [reply src/components/organisms/CardCertificate/CardCertificate.js]: The alt text doesn't get rendered correctly on all browsers (Safari), and some will show the broken image icon (Chrome).

**Pull request merged**